docs/intro/quickstart/dynamic.rst

```diff
@@ -0,0 +1,136 @@
+.. _ref_quickstart_dynamic:
+
+===============
+Dynamic Queries
+===============
+
+When updating data, we often want to modify only specific fields while leaving others unchanged. For example, we might want to update just the front text of a flashcard or only the description of a deck. There are two main approaches to handle these partial updates:
+
+1. Write a single complex query that conditionally handles optional parameters
+2. Build the query dynamically in the application code based on which fields need updating
+
+The second approach using dynamic queries tends to be more performant and maintainable. EdgeDB's TypeScript query builder excels at this use case. It allows you to construct queries dynamically while maintaining full type safety. Let's see how to implement this pattern.
```

Comment:
```suggestion
The second approach using dynamic queries tends to be more performant and maintainable. Gel's TypeScript query builder excels at this use case. It allows you to construct queries dynamically while maintaining full type safety. Let's see how to implement this pattern.
```

Reply:
Removed the separate "dynamic queries" section. This content mostly moved into the "Working with the data" section.